mod/dfrn_request.php

```diff
@@ -433,10 +433,10 @@ function dfrn_request_post(App $a)
 			}
 
 			// "Homecoming" - send the requestor back to their site to record the introduction.
-			$dfrn_url = bin2hex(System::baseUrl() . '/profile/' . $nickname);
```

Comment:
I'm not sure this should be changed.

Reply:
OK, I'll replace it with `$a->getBaseURL()` instead.